critiquebrainz/frontend/external/musicbrainz_db/artist.py

```diff
@@ -15,9 +16,14 @@ def get_artist_by_id(mbid):
     key = cache.gen_key(mbid)
     artist = cache.get(key)
     if not artist:
-        artist = fetch_multiple_artists(
+        multiple_artists = fetch_multiple_artists(
             [mbid],
             includes=['artist-rels', 'url-rels'],
+        )
+        artist = deleted_entities_to_unknown(
```

Comment:
We should do this, only if the returned artist in the call to `fetch_multiple_artists` returns empty dictionary. That makes the code much easy to read and intuitive.